components/brave_ads/core/internal/sanitize/sanitize_util.cc

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_ads/core/internal/sanitize/sanitize_util.h"
+
+#include "brave/components/text_sanitize/rs/src/lib.rs.h"
+
+namespace brave_ads {
+
+std::string SanitizeHtmlContent(const std::string& text) {
```

Comment:
why do we still need this wrapper? Returning CXXString will give you std::string automatically.

Reply:
unfortunately returning CxxString gives an error:

```
error[cxxbridge]: returning C++ string by value is not supported
   ┌─ ../../brave/components/text_sanitize/rs/src/lib.rs:11:70
   │
11 │         fn strip_non_alphanumeric_or_ascii_characters(text: &str) -> CxxString;
   │                                                                      ^^^^^^^^^ returning C++ string by value is not supported
   │
   = hint: wrap it in a UniquePtr<>
```

Reply:
ok so as discussed in dm I guess this isn't happening by default in cxx because you may not want to copy the string (which happens when doing the implicit conversion to std::string). So in that case I think we should create a cpp helper in text_sanitize that does the static cast

Reply:
Done. Added `text_sanitize_util.[h|cc]`